freqtrade/rpc/rpc_manager.py

```diff
@@ -85,14 +85,16 @@ def startup_messages(self, config: Dict[str, Any], pairlist, protections) -> Non
         timeframe = config['timeframe']
         exchange_name = config['exchange']['name']
         strategy_name = config.get('strategy', '')
+        pos_adjust_enabled = 'On' if config['position_adjustment_enable'] else 'Off'
         self.send_msg({
             'type': RPCMessageType.STARTUP,
             'status': f'*Exchange:* `{exchange_name}`\n'
                       f'*Stake per trade:* `{stake_amount} {stake_currency}`\n'
                       f'*Minimum ROI:* `{minimal_roi}`\n'
                       f'*{"Trailing " if trailing_stop else ""}Stoploss:* `{stoploss}`\n'
                       f'*Timeframe:* `{timeframe}`\n'
-                      f'*Strategy:* `{strategy_name}`'
+                      f'*Strategy:* `{strategy_name}`\n'
+                      f'*Position adjustment:* `{pos_adjust_enabled}`'
```

Comment:
From a location perspective, i'd probably move this to "above" the Timeframe line.

It somehow goes in line with minimum_roi and stoploss to me

Reply:
Done